packages/server/utils/parser.ts

```diff
@@ -23,17 +18,20 @@ export const parseForm = (buffer: Buffer | string, headers: ALBEventHeaders) =>
     let form = {} as ParsedFormDataOptions;
 
     bb.on('file', (fieldname: string, file: Readable, fileinfo: FileInfo) => {
-      const temp: FormData = { file: [], fieldname: '' };
+      const chunks: Buffer[] = [];
       file.on('data', (data: Buffer) => {
-        temp.file.push(data);
+        log.info(`Received ${data.length} bytes for field ${fieldname}`);
+        chunks.push(data);
       });
 
       file.on('end', () => {
-        temp.file = Buffer.concat(temp.file);
-        temp.fieldname = fieldname;
-        temp.fileinfo = fileinfo as FileInfo;
-
-        form = { ...temp };
+        log.info(`Finished receiving file for field ${fieldname}, total size: ${chunks.length} bytes`);
+        form = {
+          ...form,
+          fieldname,
+          filedata: Buffer.concat(chunks),
+          fileinfo,
+        };
         console.log('File parse finished');
```

Comment:
If we're using log.info already, this console.log could (and should) be changed into that as well.